schedule/yam/create_hdd_module_combinations_sles12sp5_default_pattern.yaml

```diff
@@ -0,0 +1,19 @@
+name: create_hdd_module_combinations_sles12sp5_default_pattern
+description:    >
+  Create support images with module combinations
+  on default patterns on sles12sp5.
```

Comment:
If you agree the previous comments, then this line is not needed.

Reply:
sure. thanks